jointjs/index.d.ts

```diff
@@ -492,13 +501,285 @@ declare namespace joint {
             }
+            interface PolygonAttrs extends dia.TextAttrs {
+                polygon?: ShapeAttrs;
+            }
+            class Polygon extends Generic {
+                constructor(attributes?: GenericAttributes<PolygonAttrs>, options?: Object)
+            }
+            interface PolylineAttrs extends dia.TextAttrs {
+                polyline?: ShapeAttrs;
+            }
+            class Polyline extends Generic {
+            }
             class Image extends Generic {
+                constructor(attributes?: GenericAttributes<dia.TextAttrs>, options?: Object)
+            }
+            interface PathAttrs extends dia.TextAttrs {
+                path?: ShapeAttrs;
             }
             class Path extends Generic {
+                constructor(attributes?: GenericAttributes<PathAttrs>, options?: Object)
             }
-            class Polygon extends Generic {
+            interface RhombusAttrs extends dia.TextAttrs {
+                path?: ShapeAttrs
             }
-            class Polyline extends Generic {
+            class Rhombus extends Generic {
+                constructor(attributes?: GenericAttributes<RhombusAttrs>, options?: Object)
+            }
+            interface TextBlockAttrs extends dia.TextAttrs {
+                rect?: ShapeAttrs;
+            }
+            class TextBlock extends Generic {
+                constructor(attributes?: GenericAttributes<TextBlockAttrs>, options?: Object)
+                updateSize(cell: dia.Cell, size: dia.Size): void;
+                updateContent(cell: dia.Cell, content: string): void;
+            }
+        }
+
+        namespace chess {
+            class KingWhite extends basic.Generic {
+            }
+            class KingBlack extends basic.Generic {
+            }
+            class QueenWhite extends basic.Generic {
+            }
+            class QueenBlack extends basic.Generic {
+            }
+            class RookWhite extends basic.Generic {
+            }
+            class RookBlack extends basic.Generic {
+            }
+            class BishopWhite extends basic.Generic {
+            }
+            class BishopBlack extends basic.Generic {
+            }
+            class KnightWhite extends basic.Generic {
+            }
+            class KnightBlack extends basic.Generic {
+            }
+            class PawnWhite extends basic.Generic {
+            }
+            class PawnBlack extends basic.Generic {
+            }
+        }
+
+        namespace devs {
+            interface ModelAttributes extends GenericAttributes<dia.SVGAttributes> {
+                inPorts?: string[];
+                outPorts?: string[];
+                ports?: Object;
+            }
+            class Model extends basic.Generic {
+                constructor(attributes?: ModelAttributes, options?: Object);
+                changeInGroup(properties: any, opt?: Object): boolean;
```

Comment:
Don't use `Object`, use `any` or `{}` instead. http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html